appmap/_implementation/recording.py

```diff
@@ -355,7 +355,18 @@ def wrapped_find_spec(find_spec, _, args, kwargs):
     if spec is not None:
         if getattr(spec.loader, "exec_module", None) is not None:
             loader = spec.loader
-            loader.exec_module = wrap_exec_module(loader.exec_module)
+            # This is kind of gross. As this comment linked to below describes, wrapt has trouble
+            # identifying methods decorated with @staticmethod. It offers two suggested fixes:
+            # update the class definition, or patch the function in __dict__. We can't do the
+            # former, so do the latter instead.
+            #   comment in wrapt: https://github.com/GrahamDumpleton/wrapt/blob/master/src/wrapt/wrappers.py#L730
```

Comment:
This should probably use a permalink with commit id. You can easily get one by pressing the 'Y' key in github.

Reply:
Yes, you're right. GH even did the right thing when I copied the link, and I changed it `master`. Oops. I'll change it back.